### PR TITLE
Nouveau choix: haricot géant

### PR DESCRIPTION
**Pull request by RaphaelSwift**
Please review this, @casey-collab

### CODE REVIEW

File: french/manger/trou/trou.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ Que faites-vous?
 [Je demande au crapaud de m'indiquer la sortie](../../feu-de-camp.md)
 
 [Je m'envole sur une libellule](envol/libellule.md)
+
+[Je grimpe sur un haricot géant](grimpe/haricot.md)
```
Comment (casey-collab) [inline french/manger/trou/trou.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RaphaelSwift** [comment]
I added a new subdirectory "grimpe" on my local disk and created a new file "haricot.md" in it, which links to "trou.md". However, it wasn't added automatically to my git repository, running git status didn't detect the modification. So I had to commit it and push it. What is the proper way to do it ?

**Comment by RaphaelSwift** [comment]
Please review this, @casey-collab

File: french/manger/trou/grimpe/haricot.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Vous arrivez en haut de l'haricot.
```
Comment (casey-collab) [inline french/manger/trou/grimpe/haricot.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `french/manger/trou/grimpe/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RaphaelSwift** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :hibiscus:

**Pull request merged**